cdap-ui/app/hydrator/controllers/detail/top-panel-ctrl.js

```diff
@@ -283,6 +283,21 @@ class HydratorDetailTopPanelController {
     delete exportConfig.__ui__;
     this.myPipelineExportModalService.show(exportConfig, exportConfig);
   }
+  getDeleteMessage() {
+    let triggeredPipelines = window.CaskCommon.TriggeredPipelineStore.getState().triggered.triggeredPipelines;
+    let count = triggeredPipelines.length;
+
+    if (count > 0) {
```

Comment:
Might be worth to make the final backend call to check for a list of pipelines current one will trigger. If the user deletes Pipeline1 within a UI session when in parallel someone added it as trigger can be caught. Just to stay upto-date with backend before deleting.